ckanext/ytp/comments/plugin.py

```diff
@@ -54,7 +54,10 @@ def get_helpers(self):
             'user_can_manage_comments': helpers.user_can_manage_comments,
             'get_org_id': helpers.get_org_id,
             'user_comment_follow_mute_status': notification_helpers.get_user_comment_follow_mute_status,
-            'ytp_comments_show_comments_tab_page': helpers.show_comments_tab_page
+            'ytp_comments_show_comments_tab_page': helpers.show_comments_tab_page,
+            'comment_notification_recipients_enabled':
```

Comment:
This would double up with ckanext-data-qld until the latter is updated, will that cause problems?

Reply:
we would need to do a 'lock-step' deployment with ytp and the theme to remove/add together. (or alt change its names)

Reply:
I suggest using a slightly different name here than we used in ckanext-data-qld so they don't clash. Lock-step deployments are not ideal.

Reply:
done, add ytp_comments to the new helpers.